tripal/includes/tripal.bulk_update.inc

```diff
@@ -18,11 +18,12 @@ function tripal_update_all_urls_and_titles($bundle_id, $update, $type) {
   $num_entities = $entities->rowCount();
   
   // Parse the $update variable for tokens and load those tokens.
-  preg_match_all("'/\[.*?\]/'", $update, $bundle_tokens);
+  preg_match_all("/\[.*?\]/", $update, $bundle_tokens);
 
   $fields = array();
+  print_r($bundle_tokens);
```

Comment:
this looks like the debugging variable, lets remove it right @spficklin ?

Reply:
Oh right!  Good catch.